python-avd/pyavd/_eos_designs/shared_utils/inband_management.py

```diff
@@ -62,15 +62,9 @@ def inband_mgmt_gateway(self: SharedUtilsProtocol) -> str | None:
 
         Otherwise return None
         """
-        if not self.inband_mgmt_ip:
-            return None
-
         if not self.configure_parent_for_inband_mgmt:
             return self.node_config.inband_mgmt_gateway
```

Comment:
```suggestion
        if self.node_config.inband_mgmt_ip:
            return self.node_config.inband_mgmt_gateway
```
Can we modify like this?